modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionContainer/ThenContainer.tsx

```diff
@@ -75,6 +75,9 @@ export function ThenContainer({
 		CustomItem<number>[]
 	>([]);
 
+	const [selectedObjectDefinition, setSelectedObjectDefinition] = useState(
```

Comment:
@Manda-vit Could you reorder the declarations of useStates and useMemos here by alphabetical order? Thanks :)

Reply:
@naraandrade I tried it before and the Format Source put in this order.